src/desktop/apps/auction/queries/sale_v2.js

```diff
@@ -0,0 +1,77 @@
+export default function SaleQuery(id) {
+  return `
+  query SaleQueryV2 {
+    sale(id: "${id}") {
+      _id: internalID
+      id: slug
+      associated_sale:associatedSale {
+        cover_image:coverImage {
+          cropped(width: 260, height: 110) {
+            url
+          }
+        }
+        end_at:endAt
+        href
+        id
+        is_closed:isClosed
+        is_live_open:isLiveOpen
+        is_preview:isPreview
+        live_start_at:liveStartAt
+        name
+        start_at:startAt
+      }
+      auction_state:status
+      cover_image:coverImage {
+        cropped(width: 1800, height: 600, version: "wide") {
+          url
+        }
+      }
+      currency
+      description
+      eligible_sale_artworks_count:eligibleSaleArtworksCount
+      end_at:endAt
+      is_auction:isAuction
+      is_closed:isClosed
+      is_live_open:isLiveOpen
+      is_open:isOpen
+      live_start_at:liveStartAt
+      name
+      promoted_sale:promotedSale {
+        id
+        name
+        sale_artworks:saleArtworksConnection {
```

Comment:
We can't directly pass in `sale_artworks.edges.node` since the implementation expects `sale_artworks.artworks[]`. Here is a conversion that I came up with to make it work:

https://github.com/artsy/force/pull/5392/files#diff-f1120c85eda36464dccd49e43728e8a9R36-R41

Reply:
@yuki24 will make this change as part of https://github.com/artsy/force/pull/5392 once this is merged.